src/plugins/data_views/server/rest_api_routes/public/swap_references.ts

```diff
@@ -70,6 +70,11 @@ export const swapReferencesRoute =
     router.versioned.post({ path, access: 'public', description }).addVersion(
       {
         version: INITIAL_REST_VERSION,
+        security: {
+          authz: {
+            requiredPrivileges: ['indexPatterns:manage'],
```

Comment:
I wonder if we should just rely on SO client authorization for this endpoint too? It's in the data views namespace but I don't think it's actually specific to data views and can be used for swapping any SO references.

Reply:
Updated & added a comment.